package.json

```diff
@@ -4,6 +4,7 @@
   "main": "lib/cjs/index.js",
   "module": "lib/mjs/index.js",
   "types": "lib/mjs/index.d.ts",
+  "type": "module",
```

Comment:
I got this current solution from this blog post: https://www.sensedeep.com/blog/posts/2021/how-to-create-single-source-npm-module.html

It says to leave out the "type" on purpose. The package.json files in the mjs and cjs subdirectories include the respective type parameters. (The `fixup` script creates these)

In a previous effort, I had tried to manually add the ".js" extensions like you did but got stuck on the tests. So I switched to the current version. It does work for me, at least in the node version I'm using (16.4). Although I'm still not sure why it works, tbh.

So my question is, do you need to add the "type" parameter in your environment. In theory the "exports" property gets Node to look in the correct spot.

Reply:
Ah ok, I didn't encounter any issues when I tested out Commonjs and ESM with it in there, but it works just as well without it so I'll take it out. 

My only motivation for the change was that the project itself needed to be written in ESM (from this part of the post)

> Author your code in ES6, ES-Next or Typescript using import and export.
> From this base, you can import either ES modules or CommonJS modules using import. The reverse is not true. If you author in CommonJS you cannot easily consume ES modules.


so I figured it should have the `"type"` specified